sys/src/libacpi/olsneracpi.c

```diff
@@ -234,6 +234,7 @@ ACPI_STATUS FindIOAPICs(int *pic_mode) {
 	return AE_OK;
 }
 
+#if 0
```

Comment:
They should be removed,  but not as part of this PR. The rule is you keep PRs small and focused. Makes it easier to revert them later.